openslides/agenda/projector.py

```diff
@@ -64,3 +64,32 @@ def need_full_update_for_this(self, collection_element):
         # Full update if item changes because then we may have new speakers
         # and therefor need new users.
         return collection_element.collection_string == Item.get_collection_string()
+
+
+class CurrentListOfSpeakersSlide(ProjectorElement):
+    """
+    Slide for the current list of speakers.
+
+    Nothing special to check.
+    """
+    name = 'agenda/current-list-of-speakers'
+
+    def get_requirements(self, config_entry):
+        pk = config['projector_currentListOfSpeakers_reference']
```

Comment:
New Approach: Every projector has a setting 'Current list of speakers reference' and this could be set per projector in the manage view. But also the global selector (like now) is available to set the reference from all projectors.